setup.py

```diff
@@ -52,7 +53,8 @@ def pkg_config_version_check(pkg, version):
 
 
 def pkg_config_parse(opt, pkg):
-    command = ["pkg-config", opt, pkg]
+    pkg_config = os.environ.get('PKG_CONFIG','pkg-config')
```

Comment:
Same comment as above, this wants a space before the second .get param:

`pkg_config = os.environ.get('PKG_CONFIG', 'pkg-config')`